### ISSUE TITLE
Required tag doesn't work with nested object property

### ISSUE THREAD

**Issue by myoshi1012** [swaggo/swag#504]
I define this struct and execute swag,
```
type Owner struct {
	Id        uint64   `json:"id" validate:"required"`
	Company   *Company `json:"company" validate:"required"`
}
```
but swag doesn't generate correctly.
```
  model.Owner:
    properties:
      company:
        $ref: '#/definitions/model.Company'
        type: object
      id:
        type: integer
    required:
    - id
    type: object
```

**Expected behavior**
expect:
```
  model.Owner:
    properties:
      company:
        $ref: '#/definitions/model.Company'
        type: object
      id:
        type: integer
    required:
    - company
    - id
    type: object
```

**Your swag version**
v1.6.2

**Your go version**
1.12.5

**Comment by LennyPenny** [comment]
Can this be closed now @myoshi1012 ?

**Comment by easonlin404** [comment]
fixed by #541 . Please use the latest version against the master. Reopen it if you still have any problems.
